src/Tribe/Settings_Tab.php

```diff
@@ -349,6 +366,147 @@ public function get_priority(): string {
 		return $this->priority;
 	}
 
+	/**
+	 * Determine the field value.
+	 *
+	 * @since TBD
+	 *
+	 * @param array  $field The field array.
+	 * @param string $key   The field key.
+	 *
+	 * @return mixed The field value.
+	 */
+	protected function get_field_value( $field, $key ) {
```

Comment:
I thought we talked about moving this into `Field.php`?

Reply:
Yes we did 🤦🏻